packages/logger-plugin/src/log-writer.ts

```diff
@@ -8,11 +8,12 @@ export class LogWriter {
   }
 
   startGroup(message: string) {
-    const startGroupFn = this.options.collapsed
-      ? this.logger.groupCollapsed
-      : this.logger.group;
     try {
```

Comment:
why we need use try-catch it?

Reply:
I'm not sure. It was in the original code, so I didn't want to mess with something I don't know the reason for.

Reply:
ok